tests/integration/telemetry/dashboard_test.go

```diff
@@ -45,9 +46,10 @@ import (
 
 var (
 	dashboards = []struct {
-		configmap string
-		name      string
-		excluded  []string
+		configmap                       string
+		name                            string
+		excluded                        []string
+		skipValidateOnNonPrimaryCluster bool
```

Comment:
nit: i might call this something "simpler" like `primaryClusterOnly` or `requirePrimary`.

The check would then be:

```
if !cl.IsPrimary() && d.primaryClusterOnly {
```

Or:

```
if !cl.IsPrimary() && d.requirePrimary {
```

Reply:
Thanks.. FIxed